components/text-input.json

```diff
@@ -0,0 +1,42 @@
+{ 
+  "textInput": {
```

Comment:
I had a debate with Colin on my team about this - can we use "textinput" for the component name please (i.e. neither camelcase nor kebab-case)

Reply:
sure i'll update